grumphp.yml.dist

```diff
@@ -1,9 +1,2 @@
 imports:
-  - { resource: vendor/openeuropa/code-review/dist/base-conventions.yml }
-parameters:
-  tasks.phpcs.standard: PSR2
```

Comment:
Why are we changing the value of the standard `PSR2` to `~`?
Default value https://github.com/openeuropa/code-review/blob/1.x/dist/base-conventions.yml#L13